node/pkg/governor/mainnet_chains.go

```diff
@@ -34,6 +34,7 @@ func chainList() []chainConfigEntry {
 		{emitterChainID: vaa.ChainIDOptimism, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
 		{emitterChainID: vaa.ChainIDXpla, dailyLimit: 500_000, bigTransactionSize: 50_000},
 		{emitterChainID: vaa.ChainIDBase, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		{emitterChainID: vaa.ChainIDScroll, dailyLimit: 500_000, bigTransactionSize: 50_000}, // What should these values be?
```

Comment:
@djb15 @claudijd and I discussed this. We were going to do 500k `dailyLimit` and like to do 1/10th of the daily for the `bigTransactionSize`, so this is perfect!

```suggestion
		{emitterChainID: vaa.ChainIDScroll, dailyLimit: 500_000, bigTransactionSize: 50_000},
```